bgpd/bgp_route.c

```diff
@@ -8175,7 +8175,7 @@ DEFUN (show_ip_bgp_ipv4,
 
 ALIAS (show_ip_bgp_ipv4,
        show_bgp_ipv4_safi_cmd,
-       "show bgp ipv4 (unicast|multicast) {json}",
+       "show bgp ipv4 "BGP_SAFI_CMD_STR" {json}",
        SHOW_STR
        BGP_STR
        "Address family\n"
```

Comment:
Any objections to combining the first two commits?
commit f53a268725f8c2190ee758f9dfdea943edd0da57
Author: Lou Berger <lberger@labn.net>
Date:   Sat Jan 7 17:49:01 2017 -0500

    bgpd: branch specific afi/safi changes (Issue #14)
    
    Signed-off-by: Lou Berger <lberger@labn.net>

 bgpd/bgp_route.c | 341 +++++++++++++++++----------------------------
 bgpd/bgp_vty.c   | 433 +++++++++++++++++++++-------------------------------------
 bgpd/bgp_vty.h   |   6 -
 3 files changed, 284 insertions(+), 496 deletions(-)

commit b6246ff2d96cf5cb5b2f71acea1f63566aac6f61
Author: Lou Berger <lberger@labn.net>
Date:   Fri Jan 6 10:50:31 2017 -0500

    bgpd: restore some missing afi/safi commands (Issue #14)
          Added defines and parse utility functions
          Fix vty code that treated vpn&encap as AFIs
          and some other related vty printing/handline issues
    
    Signed-off-by: Lou Berger <lberger@labn.net>

 bgpd/bgp_route.c | 154 ++++++++------------
 bgpd/bgp_vty.c   | 443 ++++++++++++++++++++++++++++++----------------------------
 bgpd/bgp_vty.h   |  23 ++-
 3 files changed, 307 insertions(+), 313 deletions(-)